test/controllers/licences.controller.test.js

```diff
@@ -148,4 +150,42 @@ describe('Licences controller', () => {
       })
     })
   })
+
+  describe('GET /licences/{id}/', () => {
+    beforeEach(async () => {
+      options = {
+        method: 'GET',
+        url: '/licences/7861814c-ca19-43f2-be11-3c612f0d744b',
+        auth: {
+          strategy: 'session',
+          credentials: { scope: ['billing'] }
+        }
+      }
+    })
+
+    describe('when a request is valid', () => {
+      beforeEach(async () => {
+        Sinon.stub(ViewLicenceService, 'go').resolves(_viewLicenceSummary())
+      })
+
+      it('returns the page successfully', async () => {
+        const response = await server.inject(options)
+
+        expect(response.statusCode).to.equal(200)
+        expect(response.payload).to.contain('Summary')
+        expect(response.payload).to.contain('Effective from')
+        expect(response.payload).to.contain('End date')
+      })
+    })
+
+    function _viewLicenceSummary () {
+      return {
+        id: '7861814c-ca19-43f2-be11-3c612f0d744b',
+        licenceRef: '01/130/R01',
+        region: 'Southern',
+        startDate: formatLongDate(new Date('2022-11-01')),
+        endDate: formatLongDate(new Date('2032-11-01'))
```

Comment:
I think if we are hard-coding the dates we're creating (which I 💯 agree with in unit tests) we could skip a step and just set the values that will be returned.

This saves bringing in the `BasePresenter` at the top of the tests as well.

```suggestion
        startDate: '1 November 2022',
        endDate: '1 November 2032'
```